packages/doctype-tile/src/tile-doctype.ts

```diff
@@ -67,7 +67,7 @@ export class TileDoctype extends Doctype {
      */
     static async makeGenesis<T extends CeramicCommit>(params: DocParams, context: Context): Promise<T> {
         // If 'deterministic' is undefined, default to creating document uniquely
-        const unique = params.deterministic ? '0' : base64Encode(randomBytes(12))
+        const unique = params.deterministic ? '0' : uint8arrays.toString(randomBytes(12), 'base16')
```

Comment:
Why was this changed from base64 to base16?

Reply:
so true. mea culpa